src/metabase/models/recent_views.clj

```diff
@@ -287,6 +292,9 @@
                    (parent-collection-valid? model_object)
                    (ellide-archived model_object))]
     {:id model_id
+     :dashboard (when (:dashboard_id card)
+                  {:name (:dashboard_name card)
+                   :id (:dashboard_id card)})
```

Comment:
I think we should update the `(def Item` on [src/metabase/models/recent_views.clj](https://github.com/metabase/metabase/pull/47784/files?file-filters%5B%5D=.clj&file-filters%5B%5D=.yaml&show-deleted-files=true&show-viewed-files=true#diff-e08f666502a3a56bae62a303fe511ace02e4f853aecda90d384d887802d30f6f) line 163, so it's kept in sync for cards.